_episodes/01-introduction.md

```diff
@@ -3,13 +3,247 @@ title: "Introduction"
+
+### 4. Choose a pre-trained model or build a new architecture from scratch
 
+Often we can use an existing neural network instead of designing one from scratch. There are a number of well publicised networks which have been shown to perform well at certain tasks, if you know of one which already does a similar task well then it makes sense to use it. This concept can be taken one step further, somebody else might have released a pre-trained network which is already trained on similar data and that can work for our purposes. If there's a network which was trained on similar data but might need a little bit more training on our specific data then this is possible using a technique called transfer learning which we'll talk about in more detail later.
+
+If instead we decide we do want to design our own network then we need to think about how many input neurons it will have, how many hidden layers and how many outputs, what types of layers we use (we'll explore the different types later on). This will probably need some experimentation and we might have to try tweaking the network design a few times before we see acceptable results.
+
+
+### 5. Choose a loss function and metrics
+
+The loss function tells the training algorithm how far away the predicted value was from the true value. There are a number of different types of function we can use here that might be suited to different types of data and network architectures. We'll look at choosing a loss function in more detail later on.
+
+
+### 6. Train the model
+
+Before we can train the model we have one final component to decide upon, the optimizer. The optimizer is responsible for taking the output of the loss function and then applying some changes to weights within the network. We can think of the training process as like trying to gradually descend a (very jagged looking) curve on a graph (that looks more like a rocky mountain side than a smooth curve) and find the bottom of it, without knowing in advance where that bottom is. Along the way we might hit some flat or even rising parts of the curve that make it look like we've reached the bottom when we haven't, we call these local minima. Different optimizers and optimizer parameters can help prevent us getting stuck in local minima. As with the loss function the best choice may depend on the kind of data and network architecture that we're using.
```

Comment:
This is a great explanation, but I think it would help to add some visuals here. Ideally a gif that shows the optimization process, jumping around the loss landscape